padle/view/confirmation.py

```diff
@@ -0,0 +1,24 @@
+from tsutils.menu.view.closable_embed import ClosableEmbedViewState
+from tsutils.menu.components.footers import embed_footer_with_state
+from discordmenu.embed.view import EmbedView
+from discordmenu.embed.components import EmbedMain
+from discord import Color as c
```

Comment:
Why are you importing it as c?  There's nothing invalid about this, but I'm curious why you're not just using it by name.

Reply:
just copied it from online lol